ChemoTracker/components/Profile/Cell.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Text, View, Image } from 'react-native';
+import { TabNavigator } from 'react-navigation';
+import styles from '../../styles/main.js';
+import color from '../../styles/color.js';
+
+class Cell extends Component {
+  constructor(props) {
+    super(props);
+    this.title = props.title;
```

Comment:
you can instead put this in `this.state = {title: props.title, description: props.description}` if the title will change in the future or just use it directly in the component (i.e `<Text style={styles.profileDetailTitle}>{this.props.title}</Text>` if it's static and there will be no changes to it
https://facebook.github.io/react-vr/docs/components-props-and-state.html

another thing you can do if you dont like doing `this.props.title` directly is to 
```
render() {
    const { title } = this.props;
.......
    <Text style={styles.profileDetailTitle}>{title}</Text>
.......